blog/you-cant-create-a-constant-using-usememo-in-react.md

```diff
@@ -0,0 +1,58 @@
+# You can't create a constant using useMemo in React
+
+There are 3 possible options to create a constant in React: `useState`, `useMemo` and `useRef`. We're going to see which one is the best way to create real constants (not constant-like values).
```

Comment:
```suggestion
There are 3 possible options to create a real constant (not constant-like values) in React: `useState`, `useMemo` and `useRef`. Now we're going to see which one is the best.
```

Reply:
This paragraph could be rewritten in a diff order, but I think you should not repeat sentences that way